TestHelper/Info.plist

```diff
@@ -15,8 +15,8 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.25.0</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>2</string>
```

Comment:
Is this getting hard reset to `2`?

Reply:
yep, when agvtool starts incrementing.

Reply:
@frederoni that doesn't look like an `agvtool` variable. It looks like `$(CURRENT_PROJECT_VERSION)` is replaced with a constant value.

Reply:
`agvtool next-version -all` increments the value of `CFBundleVersion` which previously wasn't incrementable so it got bumped to 2.